README.md

```
@@ -3,96 +3,183 @@
-  **Architecture:** Inception-ResNet-v1 trained on CASIA-WebFace
-* Android TFLite model source used in this project:
-  **Repository:** [https://github.com/shubham0204/FaceRecognition_With_FaceNet_Android](https://github.com/shubham0204/FaceRecognition_With_FaceNet_Android)
-* The Facenet `.tflite` in our repo (`app/src/main/assets/facenet.tflite`) was sourced directly from the above repository.
-* If any developer wishes to convert a FaceNet `.pb` model to `.tflite`, refer to this conversion script:
-  **File:** [`src/train_model/tflite_convert.py`](https://github.com/davidsandberg/facenet/blob/master/src/train_model/tflite_convert.py)
+- **Android TFLite Source Used**
+  Repository: [https://github.com/shubham0204/FaceRecognition_With_FaceNet_Android](https://github.com/shubham0204/FaceRecognition_With_FaceNet_Android)
+
+- The `facenet.tflite` model located at:
+
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)